eth/config.go

```diff
@@ -61,6 +62,13 @@ var DefaultConfig = Config{
 		Blocks:     20,
 		Percentile: 60,
 	},
+	Rollup: rollup.Config{
+		TxIngestionEnable:       false,
+		TxIngestionDBHost:       "localhost",
+		TxIngestionDBPort:       5432,
+		TxIngestionDBName:       "rollup",
+		TxIngestionPollInterval: 1000 * time.Millisecond,
```

Comment:
This expression can be simplified and this value is very important to the throughput of the system